<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM PEMBAYARAN</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>381</width>
     <height>140</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="kodePesananLabel">
      <property name="text">
       <string>Kode Pesanan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_kode_pesanan"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tglBayarLabel">
      <property name="text">
       <string>Tanggal Bayar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edit_tgl_bayar"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="uangBayarLabel">
      <property name="text">
       <string>Uang Bayar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edit_uang_bayar"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="uangKembaliLabel">
      <property name="text">
       <string>Uang Kembali</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edit_uang_kembali"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="kodeBayarLabel">
      <property name="text">
       <string>Kode Bayar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit_kode_bayar"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabel_pembayaran">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCetak">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cetak</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
